README.md

```diff
@@ -1145,6 +1152,33 @@ Specifies the message_digest password. Valid value is a string.
 *Required*. Ospf area associated with this cisco_interface_ospf instance. Valid values are a string, formatted as an IP address (i.e. "0.0.0.0") or as an integer.
 
 --
+### Type: cisco_interface_portchannel
+Manages configuration of a portchannel interface instance.
+
+#### Parameters
```

Comment:
For the various properties, please update them to indicate which platforms they are supported on.

For example:

port-channel lacp graceful convergence. Valid values are true, false or 'default'. This property is not supported on (Nexus 3|7k)